src/main/java/com/amihaiemil/docker/RtNetwork.java

```diff
@@ -75,47 +80,75 @@
     @Override
     public JsonObject inspect()
         throws IOException, UnexpectedResponseException {
-        throw new UnsupportedOperationException(
-            String.join(" ",
-                "Network.inspect() is not yet implemented.",
-                "If you can contribute please",
-                "do it here: https://www.github.com/amihaiemil/docker-java-api"
-            )
-        );
+        return new Inspection(this.client, this.baseUri.toString());
     }
 
     @Override
     public void remove() throws IOException, UnexpectedResponseException {
-        throw new UnsupportedOperationException(
-            String.join(" ",
-                "Network.remove() is not yet implemented.",
-                "If you can contribute please",
-                "do it here: https://www.github.com/amihaiemil/docker-java-api"
-            )
+        final UncheckedUriBuilder uri = new UncheckedUriBuilder(
+            this.baseUri.toString()
+        );
+        final HttpDelete delete = new HttpDelete(
+            uri.build()
         );
+        try {
+            this.client.execute(
+                delete,
+                new MatchStatus(delete.getURI(), HttpStatus.SC_NO_CONTENT)
+            );
+        } finally {
+            delete.releaseConnection();
+        }
     }
 
     @Override
     public void connect(final String containerId)
         throws IOException, UnexpectedResponseException {
-        throw new UnsupportedOperationException(
-            String.join(" ",
-                "Network.connect() is not yet implemented.",
-                "If you can contribute please",
-                "do it here: https://www.github.com/amihaiemil/docker-java-api"
+        final UncheckedUriBuilder uri = new UncheckedUriBuilder(
+            this.baseUri.toString() + "/connect"
+        );
+        final HttpPost post = new HttpPost(
+            uri.build()
+        );
+        post.setEntity(
+            new StringEntity(
```

Comment:
@paulodamaso Please use `JsonObject` here instead of composing it with `String.format`. This will make the code reusable when adding more methods with optional parameters. For example, in the next step, new `connect` method can be added to support optional EndpointConfig parameter, like this: `connect(final String containerId, final JsonObject endpointConfig)`